demo/admin/src/products/ProductsGrid.tsx

```diff
@@ -236,9 +233,12 @@ const productsFragment = gql`
             id
             title
         }
-        variants {
-            image
+        colors {
             name
+            colorHexCode
+        }
+        variants {
+            id
         }
```

Comment:
This is used in copyData, but removed in onPaste, so a product's variants won't be copied, right? How should this be? Copy the relations or add no variants to the copy?

Reply:
that depends on the use case, as always :)

let's not copy it for now